src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -209,16 +208,16 @@ public void moveMovables() {
 	private boolean hitsWall(IMovable m, ICollisionArea box) {
```

Comment:
Cyclomatic complexity of this method is now very high. Can it be lowered in anyway?